### ISSUE TITLE
crash occur when running demo scripts

### ISSUE THREAD

**Issue by w111liang222** [IamCreateAI/NeoVerse#8]
python inference.py     --input_path examples/videos/robot.mp4     --trajectory tilt_up     --prompt "A two-arm robot assembles parts in front of a table."     --output_path outputs/tilt_up.mp4

Loading model from models...
Loading models from: models/NeoVerse/reconstructor.ckpt
    model_name: reconstructor model_class: WorldMirror
        This model is initialized with extra kwargs: {'enable_norm': False, 'strict_load': True}
    The following models are loaded: ['reconstructor'].
Loading models from: ['models/NeoVerse/diffusion_pytorch_model-00003-of-00006.safetensors', 'models/NeoVerse/diffusion_pytorch_model-00004-of-00006.safetensors', 'models/NeoVerse/diffusion_pytorch_model-00006-of-00006.safetensors', 'models/NeoVerse/diffusion_pytorch_model-00005-of-00006.safetensors', 'models/NeoVerse/diffusion_pytorch_model-00001-of-00006.safetensors', 'models/NeoVerse/diffusion_pytorch_model-00002-of-00006.safetensors']
    model_name: wan_video_dit model_class: WanModel
        This model is initialized with extra kwargs: {'has_image_input': False, 'patch_size': [1, 2, 2], 'in_dim': 16, 'dim': 5120, 'ffn_dim': 13824, 'freq_dim': 256, 'text_dim': 4096, 'out_dim': 16, 'num_heads': 40, 'num_layers': 40, 'eps': 1e-06}

    model_name: wan_video_neoverse_controller model_class: NeoVerseControlBranch
        This model is initialized with extra kwargs: {'control_layers': (0, 5, 10, 15, 20, 25, 30, 35), 'control_in_dim': 96, 'patch_size': (1, 2, 2), 'has_image_input': False, 'dim': 5120, 'num_heads': 40, 'ffn_dim': 13824, 'eps': 1e-06}
    The following models are loaded: ['wan_video_dit', 'wan_video_neoverse_controller'].
Loading models from: models/NeoVerse/models_t5_umt5-xxl-enc-bf16.pth
    model_name: wan_video_text_encoder model_class: WanTextEncoder
    The following models are loaded: ['wan_video_text_encoder'].
Loading models from: models/NeoVerse/Wan2.1_VAE.pth
    model_name: wan_video_vae model_class: WanVideoVAE
    The following models are loaded: ['wan_video_vae'].
Using wan_video_text_encoder from models/NeoVerse/models_t5_umt5-xxl-enc-bf16.pth.
Using wan_video_dit from ['models/NeoVerse/diffusion_pytorch_model-00003-of-00006.safetensors', 'models/NeoVerse/diffusion_pytorch_model-00004-of-00006.safetensors', 'models/NeoVerse/diffusion_pytorch_model-00006-of-00006.safetensors', 'models/NeoVerse/diffusion_pytorch_model-00005-of-00006.safetensors', 'models/NeoVerse/diffusion_pytorch_model-00001-of-00006.safetensors', 'models/NeoVerse/diffusion_pytorch_model-00002-of-00006.safetensors'].
Using wan_video_vae from models/NeoVerse/Wan2.1_VAE.pth.
Using wan_video_neoverse_controller from ['models/NeoVerse/diffusion_pytorch_model-00003-of-00006.safetensors', 'models/NeoVerse/diffusion_pytorch_model-00004-of-00006.safetensors', 'models/NeoVerse/diffusion_pytorch_model-00006-of-00006.safetensors', 'models/NeoVerse/diffusion_pytorch_model-00005-of-00006.safetensors', 'models/NeoVerse/diffusion_pytorch_model-00001-of-00006.safetensors', 'models/NeoVerse/diffusion_pytorch_model-00002-of-00006.safetensors'].
Using reconstructor from models/NeoVerse/reconstructor.ckpt.
1054 tensors are updated by LoRA.
Loaded LoRA from models/NeoVerse/loras/Wan21_T2V_14B_lightx2v_cfg_step_distill_lora_rank64.safetensors
Model loaded!
Loading video from examples/videos/robot.mp4...
Generating with trajectory: tilt_up (mode=relative)
  0%|                                                                                                                                                          | 0/4 [00:00<?, ?it/s]Floating point exception (core dumped)

**Comment by YuxueYang1204** [comment]
Thanks for reporting. The "Floating point exception (core dumped)" is typically caused by a CUDA kernel crash, which is often environment-related. Could you share your environment details?
- GPU model
- CUDA version (`nvcc --version`)
- PyTorch version (`python -c "import torch; print(torch.__version__)"`)
- NVIDIA driver version (`nvidia-smi`)

We recommend using GPUs with native BF16 support (NVIDIA Ampere or newer). In the meantime, we've also added a `--low_vram` mode with ~38 GB peak GPU memory.

**Comment by zhengsyao** [comment]
It's related to a bug of cuBLAS on newer hopper gpus (H series). 
update to cuda 12.4 and related torch packages will solve the problem:
pip install torch==2.4.1 torchvision==0.19.1 --index-url https://download.pytorch.org/whl/cu124
